subsys/net/lib/openthread/platform/logging.c

```diff
@@ -44,7 +49,6 @@ void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion,
 	case OT_LOG_LEVEL_WARN:
 		LOG_WRN("%s", log_strdup(logString));
 		break;
-	case OT_LOG_LEVEL_NOTE:
```

Comment:
why was OT_LOG_LEVEL_NOTE removed?

Reply:
Actually both changes are not mine and I don't know how they get there. I think that I did something wrong during cherry picking of file revision, but it is fixed now